pages/about.html

```diff
@@ -4,9 +4,61 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About | Uyen Vong</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/about.css">
```

Comment:
Nice! The way the CSS cascades, the styles in `about.css` get applied on top of `style.css`!